code/obj/item/gun/ammo.dm

```diff
@@ -930,6 +970,7 @@
 /obj/item/ammo/bullets/antisingularity
 	sname = "Singularity buster rocket"
 	name = "Singularity buster rocket"
+	desc = "An experimental rocket containing an energy payload designed to collapse singularities."
```

Comment:
Minor-
Would appreciate something about how there are pretty awful versus humans, walls, etc etc.
I see people trying to use these to shoot people pretty often which Doesn't Work.
"Significantly less effective against non-singularities." maybe.